src/Host/Client/Test/DataInspection/ValuesTest.cs

```diff
@@ -31,13 +31,14 @@ public class ValuesTest : IAsyncLifetime {
         private const REvaluationResultProperties AllFields = unchecked((REvaluationResultProperties)~0);
 
         private readonly MethodInfo _testMethod;
+        private readonly IRHostBrokerConnector _brokerConnector = new RHostBrokerConnector(name: nameof(ValuesTest));
```

Comment:
You mean InitializeAsync, or the actual constructor? If the latter, why is that preferable?

---
In reply to: [74972244](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74972244) [](ancestors = 74972244)